### DISCUSSION TITLE
Correct way to use translations in SSR

### DISCUSSION THREAD

**Discussion by bostrom** [i18next/next-i18next#1643] [Q&A] [answered]
Hi, I have a use case where I need (or at least want) to translate some content already in the static rendering phase. As per the docs, I'm loading the translations successfully with `serverSideTranslations` inside `getStaticProps`, which gives me an object containing all translations in the requested namespaces. 

Is there any other or better way to consume these translations than referencing the object itself, something looking more like the `t` function usage? Or is the below snippet the only way? Or is it even an anti-pattern and should be avoided?

```typescript
const i18nProps = await serverSideTranslations(locale, [
  'my_namespace'
]);

const someThingWithTranslation = {
  value: 123,
  label: i18nProps._nextI18Next.initialI18nStore[locale].my_namespace.some_key
}

return {
  props: {
    someThingWithTranslation
  }
}
```

**Comment by isaachinman** [comment, marked as answer]
Are you able to consume the global i18n instance?

```tsx
import { i18n } from 'next-i18next'

i18n.t('my-namespace:my-key')
```

**Reply by bostrom** [reply]
Yes, that actually seems to work, thanks.
